### ISSUE TITLE
Password confirmation doesn't work

### ISSUE THREAD

**Issue by macak99** [victorjonsson/jQuery-Form-Validator#286]
This is form:

		<div id="register-form">
			<form action="register.php" method="post" class="toggle-disabled">
				<div class="field">
					<label for="username">Username:</label>
					<input type="text" name="Username" id="username" value="<?php echo escape(Input::get('username')); ?>" data-validation="length alphanumeric" data-validation-length="6-20">
				</div>

				<div class="field">
					<label for="password">Choose a password:</label>
					<input type="password" name="Password_confirmation" id="password" value="<?php echo escape(Input::get('password')); ?>" data-validation="length" data-validation-length="6-20">
				</div>

				<div class="field">
					<label for="password_again">Enter your password again:</label>
					<input type="password" name="Password" id="password_again" value="<?php echo escape(Input::get('password_again')); ?>" data-validation="confirmation" data-validation-confirm="Password_confirmation">
				</div>

				<div class="field">
					<label for="name">Enter your real name:</label>
					<input type="text" name="Name" id="name" value="<?php echo Input::get('name'); ?>" data-validation="length alphanumeric" data-validation-length="5-40">
				</div>

				<input type="hidden" name="token" value="<?php echo Token::generate(); ?>">
				<input type="submit" value="Register" name="submit" id="submit">
			</form>
			<script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
			<script src="//cdnjs.cloudflare.com/ajax/libs/jquery-form-validator/2.2.43/jquery.form-validator.min.js"></script>
			<script>
				$.validate({
					modules : 'toggleDisabled',
					disabledFormFilter : 'form.toggle-disabled',
					showErrorDialogs : true
				});
			</script>

Everything work except password confirmation. What am I doing wrong?

**Comment by victorjonsson** [comment]
The confirmation validator is part of the security model

```js
$.validate({
    modules : 'toggleDisabled, security',
    disabledFormFilter : 'form.toggle-disabled',
    showErrorDialogs : true
});
```
